planning/mission_planner/src/mission_planner/mission_planner.cpp

```diff
@@ -78,7 +78,7 @@ MissionPlanner::MissionPlanner(const rclcpp::NodeOptions & options)
 
   odometry_ = nullptr;
   sub_odometry_ = create_subscription<Odometry>(
-    "/localization/kinematic_state", rclcpp::QoS(1),
+    "/localization/absolute_odom", rclcpp::QoS(1),
```

Comment:
This is not necessary and I don't recommend hard-coding topic names.

Reply:
I added a parameter `map_odom_topic` to mission_planner so it is no longer hard-coded.